server/config/option.go

```diff
@@ -299,12 +300,12 @@ func (o *ScheduleOption) AddSchedulerCfg(tp string, args []string) {
 }
 
 // RemoveSchedulerCfg removes the scheduler configurations.
-func (o *ScheduleOption) RemoveSchedulerCfg(name string) error {
+func (o *ScheduleOption) RemoveSchedulerCfg(ctx context.Context, name string) error {
 	c := o.Load()
 	v := c.Clone()
 	for i, schedulerCfg := range v.Schedulers {
 		// To create a temporary scheduler is just used to get scheduler's name
-		tmp, err := schedule.CreateScheduler(schedulerCfg.Type, schedule.NewOperatorController(nil, nil), schedulerCfg.Args...)
+		tmp, err := schedule.CreateScheduler(schedulerCfg.Type, schedule.NewOperatorController(ctx, nil, nil), schedulerCfg.Args...)
```

Comment:
seem no necessary to pass a `ctx`,  just create a temporary schedule. Can we create a context in this function?

Reply:
In fact, e.g., we are trying to remove the balance leader scheduler, the TTL cache will leak if we create a new context here.